tubemq-server/src/main/java/org/apache/tubemq/server/common/utils/WebParameterUtils.java

```diff
@@ -631,8 +631,7 @@ public static String checkAndGetFilterConds(String inFilterConds,
                 continue;
             }
             String brokerIp =
-                    validAddressParameter("brokerIp", strBrokerIps[i],
-                            TBaseConstants.META_MAX_BROKER_IP_LENGTH, true, "");
+                    checkParamCommonRequires("brokerIp", strBrokerIps[i], true);
```

Comment:
In order to better expand, it is necessary to delete the IP value check, but users need to pay attention to the accuracy of the IP value when configuring the Broker’s IP